language_names/scraper.py

```diff
@@ -11,7 +11,7 @@
 from lxml import etree
 
 sys.path.append(os.path.join(os.path.abspath(os.path.dirname(__file__)), '../apertium_apy'))
-from util import to_alpha2_code  # noqa: E402
+from utils import to_alpha2_code  # noqa: E402
 
 html_tools_languages = set(map(to_alpha2_code, {
```

Comment:
This file still outputs SQL, not a TSV.

Reply:
Do I need to make this file output TSV? I have already scraped the data and put them in TSV files.

Reply:
Yes. That was my whole point!

Reply:
Okay I will modify scraper.py file but what about scraper-sil.py? It doens't work anymore.

Reply:
Is it easy to fix it? What's wrong with it?

Reply:
I discussed it on irc. Seems like we cannot scrap the data anymore. So I copied the data on the website manually and made a tsv file. Would that be fine? Or we need to work on scraper-sil.py

Reply:
Why not? `http://www-01.sil.org/iso639-3/iso-639-3.tab` still works for me... Why can we not scrape the data?

Reply:
Website works fine but scraping doesn't.
![Screenshot from 2019-03-27 00-44-37](https://user-images.githubusercontent.com/33171576/55026689-86fb2d00-5029-11e9-8739-baf6fdeb2301.png)
Please have a look

Reply:
Website might be blocking urllib

Reply:
That's very simple to fix. Just add a Chrome user-agent or equivalent.

Reply:
(also, in general, never send pictures of a terminal/code, just send the text instead.)

Reply:
Okay I will take care from next time